engine/config/locales/en.yml

```diff
@@ -22,4 +22,43 @@ en:
         <p>We use essential cookies to make our website work properly.</p>
         <p>We'd also like to use additional cookies to remember your settings and to understand how you use our website. This helps us improve our services for you.</p>
         <p>Some additional cookies come from other websites. These cookies help us show you things from those sites, like videos.</p>
-        <p>You can <a href="/about-us/information/how-we-use-cookies/">find out more about the cookies we use</a>.</p>
\ No newline at end of file
+        <p>You can <a href="/about-us/information/how-we-use-cookies/">find out more about the cookies we use</a>.</p>
+  cookie_preferences:
+    title: Your cookie settings
+    summary_html: >
+      <p>We add small text files called ‘cookies’ to your phone, tablet or computer when you use our website.</p>
+      <p>We use 2 types of cookies:</p>
+      <ul>
+        <li>essential cookies - we have to use these to make our website work</li>
+        <li>additional cookies - we’ll only use these if you give us your permission</li>
+      </ul>
+      <p>You can see <a href="/about-us/information/how-we-use-cookies/"> a full list of all the cookies we use</a>.</p>
+      <h2>Essential cookies</h2>
+      <p>We add essential cookies to your device as soon as you use our website. Without them our website won’t work properly.</p>
+      <p>You can block essential cookies in your browser settings, but you won’t be able to:</p>
+      <ul>
+        <li>save your preferences, like the country you live in</li>
+        <li>fill in forms on our website</li>
+        <li>use our online chat service</li>
+        <li>log in, if you work or volunteer for us</li>
+      </ul>
+      <h2>Additional cookies</h2>
+      <p>We won't add additional cookies unless you give us permission.</p>
+    form:
+      analytic_cookies_header_html: >
+        <h2>Accept or reject analytics cookies</h2>
```

Comment:
Since the analytics and video player headers are only one line that will (I guess) always live in an h2, we can add the h2 in the erb file and do just text here. But I guess having the html here makes it more consistent with the rest of the content. I don't have strong feelings about either, so up to you.